Router.Call: retry on VShardErrNameTransferIsInProgress error too (#77)
tarantool/go-vshard-router: CHANGELOG.md

* Get rid of nameToReplicasetMutex, use atomic instead.
* Add configurable pause before retrying r.Route in Router.Call method.
* Add ability to set custom dialer in InstaceInfo.

> * Router.Call: retry on VShardErrNameTransferIsInProgress error too (resolve issue #75).

Reviewer: ```suggestion
* Router.Call: retry on VShardErrNameTransferIsInProgress error as in the `vshard` module (#75).
```
